manifest.yaml

```
@@ -16,6 +16,14 @@ arguments:
     description: Enables pre-releasing via semantic-release
     schema:
       type: boolean
+  ciOptions.skipE2e:
+    description: Disables the e2e ci pipeline
```

Capitalize here and below?
```suggestion
    description: Disables the e2e CI pipeline
```